src/UserService/Startup.cs

```diff
@@ -51,6 +52,12 @@ public void ConfigureServices(IServiceCollection services)
             });
 
             services.AddControllers();
+            
+            services.AddControllers().AddJsonOptions(opt =>
+            {
+                opt.JsonSerializerOptions.Converters.Add(new JsonStringEnumConverter());
```

Comment:
Since version 3 of .net you need to use this string to convert the property from json to enum. In another way, there will be an error that .net cannot convert the received propertyName to enum